### DISCUSSION TITLE
El PBSendSurvey se puede dejar el enviado y recibido por como un $VALUE#user?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#224] [2. Questions and Answers] [answered]
Hola

Lo que necesito es lo siguiente:

1.- Tener un survey con un listado de surveys, mediante por la componente **Opción múltiple**, cuyos codes de las alternativas serán los codes de las surveys.
2.- En el momento que seleccione el **survey A** y lo envío, el sistema debería enviar ese survey para que yo lo pueda completar.
3.- Sin embargo si otro usuario selecciona el **survey A**, debería enviar ese survey para que él lo pueda editar.

Lo pregunto ya que las veces que he utilizado ese bot siempre he puesto la _id en duro, entonces si colocara: **$VALUE#user**, debería funcionar?

Adjunto imagen con los campos que estoy consultando
![image](https://user-images.githubusercontent.com/61419471/120347403-3e56f280-c2ca-11eb-8f64-1c96d11db313.png)

Gracias :>

**Comment by guillevalin** [comment, marked as answer]
@NicolasPLG enviado por y recibido por es una funcionalidad deprecada del pasado que se mantiene por compatibilidad con antiguas implementaciones. El campo importante es el "Recibido por" quien será quien puede editar y enviar el formulario.

Si $VALUE#user será quien envía el formulario, está correcto y el campo "Enviado por" puede ser cualquier ID de usuario válido (normalmente se dejaba en duro el ID del usuario de sistema).

**Reply by NicolasPLG** [reply]
hola  @guillevalin, perfecto entonces probaré con eso, cerraré el tema y s me sale algo similar lo dejo en referencia
